docs/frequently_asked_questions.md

```diff
@@ -84,7 +84,7 @@ all of Unity has been bootstrapped so that the Plugin can hook all the appropria
 ## Why does the Demo Scene fail to load?
 
 There is a standard sample pack, and several extra packs in the EOS Unity Plugin. If a scene doesn't load, remember to import the wanted extra pack.
-Additionally, make sure all wanted sample scenes are included in the build settings as shown in steps 4.-6. of <a href="/readme.md#importing-the-samples">Importing the samples</a>.
+Additionally, make sure all wanted sample scenes are included in the build settings as shown in steps 4.-6. of <a href="/README.md#importing-the-samples">Importing the samples</a>.
```

Comment:
This link doesn't take me to the "Importing the samples" section. It takes me to the Unsupported platforms section instead. 
![image](https://github.com/PlayEveryWare/eos_plugin_for_unity/assets/50968607/855b6cdc-391e-4d26-a674-d73f876a14ae)